spec/config/mongoid.rb

```diff
@@ -2,12 +2,14 @@
   config.connect_to('delayed_job_shallow_mongoid_test')
 end
 
+Mongo::Logger.logger = ::Logger.new('tmp/mongo.log') if defined?(Mongo)
```

Comment:
You should suppress that like this: https://github.com/mongoid/mongoid-slug/blob/master/spec/spec_helper.rb#L45 and not deal with log.